### ISSUE TITLE
Right side bar wrong position, big margin from right side

### ISSUE THREAD

**Issue by dimti** [RocketChat/Rocket.Chat#6710]
Rocket.Chat Version: 0.52.0
Running Instances: 1
DB Replicaset OpLog:
Node Version: 4.4.6

![_2017-04-18_06-51-07](https://cloud.githubusercontent.com/assets/582144/25114001/f9e31c46-2403-11e7-964f-eafb67caeeec.png)

**Comment by MartinSchoeler** [comment]
@dimti does this happens every time you open the side nav? And are you using any custom css?

**Comment by dimti** [comment]
Custom css no, no..
Eror still exists in brwoser and in electron app.

In logs exists some records about css processor, but may be that not the side nav:


```апр 14 16:14:15 osboxes rocketchat[5177]: {"line":"160","file":"rocketchat_migrations.js","message":"Migrations: Not migrating, already at version 88","time":{"$date":1492175655409},"level":"info"}
апр 14 16:14:15 osboxes rocketchat[5177]: { [Error: error evaluating function `fade`: color.toHSL is not a function]
апр 14 16:14:15 osboxes rocketchat[5177]:   type: 'Runtime',
апр 14 16:14:15 osboxes rocketchat[5177]:   filename: 'input',
апр 14 16:14:15 osboxes rocketchat[5177]:   index: 8753,
апр 14 16:14:15 osboxes rocketchat[5177]:   line: 375,
апр 14 16:14:15 osboxes rocketchat[5177]:   callLine: NaN,
апр 14 16:14:15 osboxes rocketchat[5177]:   callExtract: undefined,
апр 14 16:14:15 osboxes rocketchat[5177]:   column: 16,
апр 14 16:14:15 osboxes rocketchat[5177]:   extract:
апр 14 16:14:15 osboxes rocketchat[5177]:    [ '\t&:focus {',
апр 14 16:14:15 osboxes rocketchat[5177]:      '\t\tborder-color: fade(@primary-background-contrast, 50%);',
апр 14 16:14:15 osboxes rocketchat[5177]:      '\t}' ],
апр 14 16:14:15 osboxes rocketchat[5177]:   message: 'error evaluating function `fade`: color.toHSL is not a function',
апр 14 16:14:15 osboxes rocketchat[5177]:   stack: undefined }
апр 14 16:14:16 osboxes rocketchat[5177]: Updating process.env.MAIL_URL
апр 14 16:14:16 osboxes rocketchat[5177]: Using GridFS for custom sounds storage
апр 14 16:14:16 osboxes rocketchat[5177]: Using FileSystem for custom emoji storage
апр 14 16:14:16 osboxes rocketchat[5177]: ufs: temp directory created at "/tmp/ufs"
апр 14 16:14:18 osboxes rocketchat[5177]: [Fri Apr 14 2017 16:14:18 GMT+0300 (MSK)] WARNING A script has tried registering a HTTP route while the HTTP server is disabled with --disabled-httpd.
апр 14 16:14:18 osboxes rocketchat[5177]: Loaded hubot-help/src/help.coffee
апр 14 16:14:18 osboxes rocketchat[5177]: Loaded hello.coffee
апр 14 16:14:18 osboxes rocketchat[5177]: Loaded zen.coffee
апр 14 16:14:21 osboxes rocketchat[5177]: Using GridFS for Avatar storage
апр 14 16:14:22 osboxes rocketchat[5177]: ➔ System ➔ startup
```

**Comment by karlprieb** [comment]
Hey @dimti which browser do you use?

**Comment by dimti** [comment]
Google Chrome, Firefox, and on Electron similar view. An other separate account and an two many computers

**Comment by karlprieb** [comment]
@dimti can you update your instance to version 0.55 and test again?

**Comment by dimti** [comment]
Updated to 0,55.0. Right bar with big margin from right side.

Maybe problem in node v4.4.6. Used node version by default 0.10.46, but on that node version exists problems with bcrypt npm package. And after up to 0.55 (or after my handle changes in node_modules/.../bcrypt package files, i see other error in journal if started rocket chat on node  v0.10.46

```апр 20 00:45:10 osboxes rocketchat[31220]: Starting rocketchat...
апр 20 00:45:10 osboxes rocketchat[31220]: /usr/share/rocketchat-server/bundle/programs/server/mini-files.js:74
апр 20 00:45:10 osboxes rocketchat[31220]: const unicodeNormalizePath = (path) => {
апр 20 00:45:10 osboxes rocketchat[31220]:                                      ^
апр 20 00:45:10 osboxes rocketchat[31220]: SyntaxError: Unexpected token >
апр 20 00:45:10 osboxes rocketchat[31220]:     at Module._compile (module.js:439:25)
апр 20 00:45:10 osboxes rocketchat[31220]:     at Object.Module._extensions..js (module.js:474:10)
апр 20 00:45:10 osboxes rocketchat[31220]:     at Module.load (module.js:356:32)
апр 20 00:45:10 osboxes rocketchat[31220]:     at Function.Module._load (module.js:312:12)
апр 20 00:45:10 osboxes rocketchat[31220]:     at Module.require (module.js:364:17)
апр 20 00:45:10 osboxes rocketchat[31220]:     at require (module.js:380:17)
апр 20 00:45:10 osboxes rocketchat[31220]:     at Object.<anonymous> (/usr/share/rocketchat-server/bundle/programs/server/boot.js:9:13)
апр 20 00:45:10 osboxes rocketchat[31220]:     at Module._compile (module.js:456:26)
апр 20 00:45:10 osboxes rocketchat[31220]:     at Object.Module._extensions..js (module.js:474:10)
апр 20 00:45:10 osboxes rocketchat[31220]:     at Module.load (module.js:356:32)
```

On node v4.4.6 rocketchat started and in log exists some errors

```апр 20 00:45:37 osboxes systemd[1]: Started Rocket.Chat server.
апр 20 00:45:37 osboxes rocketchat[31947]: Reading configuration from /etc/rocketchat/rocketchat-server.conf
апр 20 00:45:37 osboxes rocketchat[31947]: Sourcing nvm script at /usr/share/nvm/init-nvm.sh
апр 20 00:45:38 osboxes rocketchat[31947]: Installing nodejs version 4.4.6 via nvm.
апр 20 00:45:39 osboxes rocketchat[31947]: v4.4.6 is already installed.
апр 20 00:45:40 osboxes rocketchat[31947]: Now using node v4.4.6 (npm v2.15.5)
апр 20 00:45:41 osboxes rocketchat[31947]: Now using node v4.4.6 (npm v2.15.5)
апр 20 00:45:41 osboxes rocketchat[31947]: Installing rocketchat npm dependencies.
апр 20 00:45:42 osboxes rocketchat[31947]: npm WARN package.json meteor-dev-bundle@0.0.0 No description
апр 20 00:45:42 osboxes rocketchat[31947]: npm WARN package.json meteor-dev-bundle@0.0.0 No repository field.
апр 20 00:45:42 osboxes rocketchat[31947]: npm WARN package.json meteor-dev-bundle@0.0.0 No README data
апр 20 00:45:42 osboxes rocketchat[31947]: npm WARN package.json meteor-dev-bundle@0.0.0 No license field.
апр 20 00:45:42 osboxes rocketchat[31947]: > meteor-dev-bundle@0.0.0 install /usr/share/rocketchat-server/bundle/programs/server
апр 20 00:45:42 osboxes rocketchat[31947]: > node npm-rebuild.js
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! Linux 4.10.8-1-ARCH
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! argv "/home/osboxes/.nvm/versions/node/v4.4.6/bin/node" "/home/osboxes/.nvm/versions/node/v4.4.6/bin/npm" "rebuild" "--update-binary"
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! node v4.4.6
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! npm  v2.15.5
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! path /usr/share/rocketchat-server/bundle/programs/server/npm/node_modules/meteor/emojione_emojione/node_modules/grunt/bin/grunt
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! code EPERM
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! errno -1
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! syscall chmod
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! Error: EPERM: operation not permitted, chmod '/usr/share/rocketchat-server/bundle/programs/server/npm/node_modules/meteor/emojione_emojione/node_modules/grunt/bin/grunt'
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!     at Error (native)
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!  { [Error: EPERM: operation not permitted, chmod '/usr/share/rocketchat-server/bundle/programs/server/npm/node_modules/meteor/emojione_emojione/node_modules/grunt/bin/grunt']
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!   errno: -1,
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!   code: 'EPERM',
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!   syscall: 'chmod',
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!   path: '/usr/share/rocketchat-server/bundle/programs/server/npm/node_modules/meteor/emojione_emojione/node_modules/grunt/bin/grunt' }
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! Please try running this command again as root/Administrator.
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! Please include the following file with any support request:
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!     /usr/share/rocketchat-server/bundle/programs/server/npm/node_modules/meteor/emojione_emojione/npm-debug.log
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! Linux 4.10.8-1-ARCH
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! argv "/home/osboxes/.nvm/versions/node/v4.4.6/bin/node" "/home/osboxes/.nvm/versions/node/v4.4.6/bin/npm" "install"
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! node v4.4.6
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! npm  v2.15.5
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! code ELIFECYCLE
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! meteor-dev-bundle@0.0.0 install: `node npm-rebuild.js`
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! Exit status 255
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! Failed at the meteor-dev-bundle@0.0.0 install script 'node npm-rebuild.js'.
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! This is most likely a problem with the meteor-dev-bundle package,
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! not with npm itself.
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! Tell the author that this fails on your system:
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!     node npm-rebuild.js
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! You can get information on how to open an issue for this project with:
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!     npm bugs meteor-dev-bundle
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! Or if that isn't available, you can get their info via:
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!     npm owner ls meteor-dev-bundle
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! There is likely additional logging output above.
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR! Please include the following file with any support request:
апр 20 00:45:45 osboxes rocketchat[31947]: npm ERR!     /usr/share/rocketchat-server/bundle/programs/server/npm-debug.log
апр 20 00:45:45 osboxes rocketchat[31947]: Starting rocketchat...
апр 20 00:45:51 osboxes rocketchat[31947]: Will load cache for users
```

RocketChat install on ArchLinux from aur. Systemd service file execute bash script from aur vendor. That is code rocketchat-server.sh
```#!/usr/bin/env bash

BASE_DIR=/usr/share/rocketchat-server/bundle
CONFIG_FILE=/etc/rocketchat/rocketchat-server.conf
NVM_SCRIPT=/usr/share/nvm/init-nvm.sh
NODE_VERSION=0.10.46
NODE_VERSION=4.4.6

errorout() { echo "$@" && exit 1; }

# Read rocketchat configuration.
[[ ! -r "${CONFIG_FILE}" ]] && errorout "Could not read ${CONFIG_FILE}!"
echo "Reading configuration from ${CONFIG_FILE}"
# Automatically export variables from CONFIG_FILE.
set -a
. "${CONFIG_FILE}"
set +a
[[ -z "${ROOT_URL}" ]] && errorout "No ROOT_URL found in ${CONFIG_FILE}!"

# Prepare node environment.
echo "Sourcing nvm script at ${NVM_SCRIPT}"
source "${NVM_SCRIPT}"
echo "Installing nodejs version ${NODE_VERSION} via nvm."
nvm install "${NODE_VERSION}"
nvm use "${NODE_VERSION}"

# Install dependencies.
cd "${BASE_DIR}/programs/server"
echo "Installing rocketchat npm dependencies."
npm install

# Run the rocketchat server.
echo "Starting rocketchat..."
node "${BASE_DIR}/main.js"
```

**Comment by honnel** [comment]
Same issue here, what kind of info (<img width="206" alt="bildschirmfoto 2017-04-27 um 08 14 42" src="https://cloud.githubusercontent.com/assets/142107/25470242/a0639c20-2b21-11e7-940a-fd3fe4f795bd.png">) do you need?

**Comment by MartinSchoeler** [comment]
This pr should probably have fixed that #7448 . Please reopen this if the problem persists. Thanks!
